build.gradle

```diff
@@ -20,6 +20,10 @@ ext {
     jacksonVersion = '2.10.0'
     klaxonVersion = '5.0.1'
     kotlinVersion = '1.3.61'
+    besuPluginVersion = '20.10.2'
+    besuInternalVersion = '20.10.2'
```

Comment:
Why's this defined twice? Also is it necessary to have the internal version of Besu here? Surely it should be the same as the main version. Likewise do we need to call out the crypto dependency version?

Reply:
Removed the Duplicate besuInternalVersion. 
The besu jars in the maven repos are in two different groups  and I noticed few jars were updated at different rate than the plugin-api in the past.

The crypto dependency version is needed for them to be pulled from the bintray and its not coming packaged with the other jars from maven.